src/websocket/websocket_endpoint.cpp

```diff
@@ -0,0 +1,56 @@
+#include "./websocket_endpoint.h"
+#include "../logging.h"
+
+namespace sensr
+{   
+  WebSocketEndPoint::WebSocketEndPoint() : 
+    WebSocketEndPointBase()
+  {
+    Init(endpoint_);
```

Comment:
Init seems to be called only in the constructor, so why don't you make the constructor initialize instead of this?

Reply:
Ah, it's quite difficult to explain here.
Long story to short, the exactly same codes was placed in 2 constructors. (WebSocketEndpoint:WebSocketEndpoint:, WebSocketSecureEndpoint::WebSocketSecureEndpoint)
So, I just merged them into one place. it is `Init`